daprdocs/content/en/developing-applications/integrations/Azure/azure-authentication/howto-wif.md

```diff
@@ -0,0 +1,122 @@
+---
+type: docs
+title: "How to: Configure dapr to use workload identity federation on Azure"
+linkTitle: "How to: Configure dapr to use workload identity federation on Azure"
+weight: 20000
+description: "Learn how to configure dapr to use workload identity federation on Azure."
+---
+
+This guide will help you configure your Kubernetes cluster to run dapr with Azure workload identity federation.
+
+## What is it?
+
+[Workload identity federation](https://learn.microsoft.com/entra/workload-id/workload-identities-overview) 
+is a way for your applications to authenticate to Azure without having to store or manage credentials as part of 
+your releases.
+
+By using workload identity federation, any dapr components that target Azure will be able to authenticate transparently
+with no extra configuration! 🎉
+
+### How does it differ?
+
+Workload identity federation is one of a few ways Azure offers for your applications to gain access to Azure 
+resources.  Other options include:
+
+ - [Pod Managed Identities]({{< ref howto-mi.md >}}) - [Deprecated](https://learn.microsoft.com/azure/aks/use-azure-ad-pod-identity) method for authenticating applications at a pod level.
```

Comment:
Suggest moving this into the https://github.com/dapr/docs/blob/v1.15/daprdocs/content/en/developing-applications/integrations/Azure/azure-authentication/authenticating-azure.md article as part of the overall Azure Authentication overview. 

```suggestion
 - [Pod Managed Identities]({{< ref howto-mi.md >}}) - [Deprecated](https://learn.microsoft.com/azure/aks/use-azure-ad-pod-identity) method for authenticating applications at a pod level. This is deprecated and not recommended. 
```